### PR TITLE
ACCUMULO-4479 start of upgrade text

### PR DESCRIPTION
**Pull request by mjwall**
Needs discussion.  Please provide feedback @busbey and @ctubbsii

### CODE REVIEW

File: UPGRADING.md [hunk 1]
```diff
@@ -18,6 +18,22 @@ limitations under the License.
 Upgrading
 =========
 
+# From 1.7 to 1.8
+
+Upgrades from 1.7 to 1.8 are be possible with little effort as no changes were made at the data layer and RPC changes
```
Comment (keith-turner) [inline UPGRADING.md]: `are be possible`

Comment (mjwall) [inline UPGRADING.md]: haha, I copied from the previous one.  It is there too, I'll clean them both up.

**Comment by ctubbsii** [comment]
Repeating what I said in the JIRA, I don't think it's useful to provide static upgrade instructions for specific upgrade paths, where it's prone to being out-of-date (and possibly outright incorrect). The documentation should point to the most up-to-date docs on the website, specifically the an upgrading section of the release notes, which we can create upon release, and keep updated as we find issues affecting upgrade interoperability.

I also don't think it's useful to document upgrade paths to older releases in the current one. It's a bit like if Microsoft Windows 2010 installation documentation contained upgrade instructions from Windows 95 to Windows 98. It doesn't make sense to me.

If it were entirely up to me, I'd simply delete this file and make it a point to have an UPGRADING section on every release notes from here on.

**Comment by mjwall** [comment]
@ctubbsii I am not against your suggestion.  How much do we want to do for the 1.8.1 release?

**Comment by ctubbsii** [comment]
Personally, I think it's fine to remove and use release notes for all future releases from current branches.

**Comment by mjwall** [comment]
Closing this PR in favor of removing the UPGRADING.md file and including upgrade instructions in the release notes.  See https://issues.apache.org/jira/browse/ACCUMULO-4479

**Pull request closed without merging**